frontend-react/src/config/api/types.ts

```diff
@@ -0,0 +1,475 @@
+export enum LookupTables {
+    VALUE_SET = "sender_automation_value_set",
+    VALUE_SET_ROW = "sender_automation_value_set_row",
+}
+
+export enum OverallStatus {
+    VALID = "Valid",
+    ERROR = "Error",
+    WAITING_TO_DELIVER = "Waiting to Deliver",
+}
+
+/*
+The error codes map to the error types specified in the serializer
+ */
+export enum ErrorCodeTranslation {
+    INVALID_HL7_MSG_VALIDATION = "Invalid entry for field.",
+    INVALID_MSG_PARSE_DATE = "Invalid entry for field. Reformat to either the HL7 v2.4 TS or ISO 8601 standard format.",
+    INVALID_HL7_MSG_FORMAT_INVALID = "",
+    INVALID_MSG_PARSE_TELEPHONE = "The string supplied is not a valid phone number. Reformat to a 10-digit phone number (e.g. (555) 555-5555).",
+    UNKNOWN = "",
+}
+
+declare global {
```

Comment:
Minor point, but it might be nice to spread these types spread out across a few files just to make additions and maintenance a bit easier.  We could still declare them as global types from said files -- my feeling is that they can just be grouped around concepts/domains.  Definitely not a blocker, though!  Just thinking aloud about future ideas 🤔

Reply:
The plugin is only going to be able to export a single file so this was done in an attempt to get adjusted to the idea.

Reply:
Ah, I see I see!  No worries then :)